### DISCUSSION TITLE
Pseudo class properties didn't work

### DISCUSSION THREAD

**Discussion by BiDuang** [AvaloniaUI/Avalonia#13215] [Q&A] [answered]
## Describe the bug
Defines a style pseudo class and sets style properties on it. Using `Avalonia DevTools` I found that the properties are correct but display wrong.

```axaml
            <Style Selector="Button.WarningButton">
                <Setter Property="Background" Value="#CC4D11" />
            </Style>
            <Style Selector="Button.WarningButton:pointerover">
                <Setter Property="Background" Value="#EDA482" />
            </Style>
            <Style Selector="Button.WarningButton:pressed">
                <Setter Property="Background" Value="#932107" />
            </Style>
```

![1642E0057B9B21EA864D2D6CEFE3F704](https://github.com/AvaloniaUI/Avalonia/assets/40264028/f47207d4-0bcb-4b29-80dd-804400b74969)

## Environment

- OS: Windows 11 Home Edition 22H2 22621.2283
- Avalonia-Version: 11.0.4

**Comment by maxkatz6** [comment, marked as answer]
See https://docs.avaloniaui.net/docs/next/guides/styles-and-resources/troubleshooting#selector-with-a-pseudoclass-doesnt-override-the-default

**Reply by BiDuang** [reply]
Thanks for your help.😀
